src/components/complaints/NewComplaintForm.js

```diff
@@ -0,0 +1,94 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import TextField from 'material-ui/TextField';
+
+export default class NewComplaintForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      report: '',
+      ocurrenceDate: '',
+      categoryId: '',
+      address: '',
+      latitude: '',
+      longitude: '',
+    };
+  }
+
+  onPressSaveButton() {
+    this.props.salvarDenuncia(this.state);
+  }
+
+  setProperty(event, property) {
+    const currentState = this.state;
+    this.setState({
+      ...currentState,
```

Comment:
Tu nem precisaria disso aqui. Poderia ser simplesmente 

`this.setState({ [property]: e.target.value })`